libs/application/templates/secondary-school/src/lib/messages/conclusion.ts

```diff
@@ -76,5 +76,10 @@ export const conclusion = {
         'Umsóknartímabili lýkur aa. júní. Ekki verður farið að vinna úr umsóknum fyrr en eftir þessa dagsetningu. Ef þú af einhverjum ástæðum vilt gera breytingar þarftu að gera það áður en tímabilinu lýkur.',
```

Comment:
```suggestion
        'Umsóknartímabili lýkur {date}. júní. Ekki verður farið að vinna úr umsóknum fyrr en eftir þessa dagsetningu. Ef þú af einhverjum ástæðum vilt gera breytingar þarftu að gera það áður en tímabilinu lýkur.',
```

And to insert the variable:
```
description: {
            ...m.example,
            values: { date: '13-01-25' },
          },
```

Reply:
In the first version of this application this string (and date) will be stored in contentful, in the next version we'll retrieve the date from a service (the service isn't quite ready) and make this change.